rinja_derive/src/generator.rs

```diff
@@ -172,10 +172,11 @@ impl<'a> Generator<'a> {
                 Source::Source(_) => path != &*self.input.path,
             };
             if path_is_valid {
-                let path = path.to_str().unwrap();
+                let canonical_path = path.canonicalize().unwrap();
+                let include_path = canonical_path.to_str().unwrap();
```

Comment:
Do we actually need to convert it to string? Can't we just use its `Debug` impl?

Reply:
Good idea! No, it seems to work just fine without it. And instead of panicking if the path could not be resolved, we can simply use the relative path. I think that's quite elegant. :)